Pod/Core/StepDefiner.swift

```diff
@@ -164,6 +177,19 @@ open class StepDefiner: NSObject, XCTestObservation {
             f(value)
         }
     }
+    
+    /**
+     Create a step which _exactly_ matches the passed in string abd contains one matching group to match collection of `MatchedStringRepresentable` values
+
+     Don't pass anything for file: or path: - these will be automagically filled out for you. Use it like this:
+
+     - parameter exactly: The expression to _exactly_ match against
+     - parameter f: The step definition to be run, passing in the first capture group from the expression
+     */
+    open func step<T: Collection & MatchedStringRepresentable>(exactly exact: String, file: String = #file, line: Int = #line, f: @escaping (T)->()) {
+        let regexp = "^\(exact)$"
+        step(regexp, f: f)
```

Comment:
Can we pass `file` and `line` into this method please - otherwise reported errors won't link back to the right place

Reply:
Done